optimum/utils/modeling_utils.py

```diff
@@ -0,0 +1,7 @@
+def recurse_setattr(module, name, value):
+    """A wrapper function to recursively set attributes to a module."""
```

Comment:
It does not wrap anything.

Reply:
fixed in [91a37bc](https://github.com/huggingface/optimum/pull/676/commits/91a37bcd2d8998373e5acdfd5e5231692208320b)